### PR TITLE
Add password recovery in FAQ

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

File: src/documents/en/faq.html.md [hunk 1]
```diff
@@ -30,5 +30,10 @@ We also perform daily backups of your data, so we can restore them if needed.
 
 We provide 40GB of storage with each instance (well, technically 60GB are available, but we recommend for now not to use more than 40GB)
 
+## What can i do if I forget/loose my password ?
+
+For security and confidentiality reasons, we never have access to your decrypted password, and as a result, the datas of your different apps will be lost. But happely enough, not your cozy ! The only solution is to create a new password by retyping it in the "password recovery email", that you can activate in the login page of your cozy ("Forget password? Email me instructions"). You will then have to reinitialize all your apps.
```
Comment (clochix) [inline src/documents/en/faq.html.md]: This isn't very clear and partially false. Only the encrypted data will be lost, and we don't encrypt everything, only passwords. So when user forget its password, he/she can request a new one, but this means he/she will have to re-enter the passwords in every application that use passwords, for example Emails, Konnectors, Kresus…

File: src/documents/en/faq.html.md [hunk 2]
```diff
@@ -30,5 +28,11 @@ We also perform daily backups of your data, so we can restore them if needed.
 
 We provide 40GB of storage with each instance (well, technically 60GB are available, but we recommend for now not to use more than 40GB)
 
+## What can i do if I forget my password ?
+
+The "password recovery email", that you can activate in the login page of your cozy ("Forget password? Email me instructions"), will send you a link that will enable you to enter a new password. If needed, you will have to reenter this new password in the different apps of your cozy.
```
Comment (clochix) [inline src/documents/en/faq.html.md]: Sorry but this is still wrong, this PR should'nt have been merged. You don't need to enter the new password of the server in the applications. You need to take a tour of all the applications that store a password to connect to a remote account (for example Emails, Konnectors, Kresus), and edit this account to enter again the password.

File: src/documents/fr/faq.html.md [hunk 3]
```diff
@@ -31,6 +31,10 @@ Par ailleurs, nous effectuons une sauvegarde de vos données toutes les nuits, d
 
 Nous offrons 40Go de stockage garantis pour chaque serveur (techniquement, chaque instance dispose de 60Go, mais nous vous conseillons pour l’instant de ne pas dépasser les 40Go).
 
+## Que faire si j'oublie mon mot de passe ?
 
+L'"email de réinitialisation", que vous pouvez activer dans la page de login de votre cozy ("Forget password? Email me instructions"), vous enverra un lien qui vous permettra d'entrer un nouveau mot de passe. Si besoin, il va falloir rappeler ce nouveau mot de passe dans les différentes applications de votre cozy.
```
Comment (clochix) [inline src/documents/fr/faq.html.md]: « Une fois votre mot de passe mis à jour, il vous faudra faire un tour de toutes les applications qui utilisent un mot de passe pour se connecter à un compte distant (par example Mail, Konnectors, Kresus…), et saisir à nouveau les mots de passe des comptes. Pour des raisons de sécurité, lorsque le mot de passe de votre serveur est changé, nous effaçons les différents mots de passe de vos autres comptes stockés dans Cozy. »

**Comment by frankrousseau** [comment]
@clochix can you make a PR with additional information?

**Pull request merged**